examples/test.cpp

```diff
@@ -1,6 +1,43 @@
-#include "../include/featurelib.h"
+#include "../include/feature_lib.h"
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+
+std::vector<uint8_t> matToVector(const cv::Mat &mat) {
+  std::vector<uint8_t> result;
+  cv::Mat cont = mat.isContinuous() ? mat : mat.clone();  // make sure the Mat is continuous
+  cont.reshape(1, 1).copyTo(result);  // copy from a 1D Mat
+  return result;
+}
+
+int main(int argc, char *argv[]) {
+
+  if (argc != 2) {
+    std::cerr << "Illegal CLI arguments num. Expected only filepath (including the command name) but was "
```

Comment:
`std::cerr << "Illegal CLI arguments num. Expected only a file path but was " << argc << " arguments (including the command name)"` would be better

Reply:
done